terminology.md

```diff
@@ -247,6 +247,59 @@ In Firefox's Web Console the [command line interpreter](https://developer.mozill
 - [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)
 .....
 
+### Agent
+
+#### Definition
+An Agent is a thread that holds all state within a JavaScript VM instance.
+
+### Realm
+#### Definition
+A realm consists of a set of [intrinsic](###Intrinsic) objects, an ECMAScript global environment, all of the ECMAScript code that is loaded within the scope of that global environment, and other associated state and resources (i.e. a global object and an associated set of [primordial](###Primoridal) objects). Today, in the browser, realms can be created via same origin iframes.
+#### References
+[ECMA262 Spec](https://tc39.es/ecma262/#sec-code-realms)
+
+### Intrinsic
+#### Definition
+All objects that can be observed by a program, whether it is via a global ref, or some other mechanism to access those objects, and then those that can be accessed via syntax/grammar. A reference such as %name% means the intrinsic object.
+#### Example
+`[]`
+#### Reference
+[ECMA262 Spec](https://tc39.es/ecma262/#sec-well-known-intrinsic-objects)
+
+### Primordial
+#### Definition
+Globlal, mutable objects that must exist before any code runs.
+#### Example
+`%ArrayPrototype%`
+
+### Normative
+#### Definition
+The existing way something is done (i.e. status quo)
```

Comment:
Nope. Normative statements are those that constrain the observable behavior of implementations (i.e., the behavior that conforming implementations are allowed to exhibit).

Reply:
Thanks!